server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -117,6 +117,21 @@ func (s *BuildBuddyServer) CreateUser(ctx context.Context, req *uspb.CreateUserR
 	}, nil
 }
 
+func (s *BuildBuddyServer) GetGroup(ctx context.Context, req *grpb.GetGroupRequest) (*grpb.GetGroupResponse, error) {
```

Comment:
worth testing this with both enterprise and OS -- i suspect you'd get a NPE here on OS because userDB is not present in that config.

Maybe add a check like:

if userDB == nil {
		return nil, status.UnimplementedError("Not Implemented")
	}

Reply:
Done